operations/helm/ct.yaml

```diff
@@ -4,11 +4,8 @@ target-branch: main
 chart-dirs:
   - operations/helm/charts
 chart-repos:
-  - grafana=https://grafana.github.io/helm-charts
-  - prometheus-community=https://prometheus-community.github.io/helm-charts
-  - elastic=https://helm.elastic.co
```

Comment:
I assume these were taken in as part of the copy from helm-charts and now you're just cleaning them up? as well as the hashicorp below

Reply:
yes, correct, going to add that to the squashed commit message!